examples/groth16-verifier/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "groth16-verifier"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+methods = { path = "methods" }
+risc0-groth16 = { path = "groth16" }
+risc0-zkvm = { path = "../../risc0/zkvm" }
+serde_json = "1.0"
+
+# Always optimize; building and running the guest takes much longer without optimization.
+[profile.dev]
+opt-level = 3
+
+[profile.dev.build-override]
+opt-level = 3
+
+[profile.release]
+debug = 1
+lto = true
+
+[profile.release.build-override]
+opt-level = 3
```

Comment:
The workspace already contains all of this. They will be ignored when specified here.